qds_sdk/commands.py

```diff
@@ -87,10 +87,26 @@ def run(cls, **kwargs):
         Returns:
             Command object
         """
+        err_pointer, tmp_pointer, new_bytes = 0, 0, 0
         cmd = cls.create(**kwargs)
         while not Command.is_done(cmd.status):
```

Comment:
It could be possible that command is done but in the previous iteration we didn't get full remaining logs as it was still running. So it would be good to fetch the logs once again and apply the same logic as in while loop. We have seen such cases in UI.